frontend/src/index.css

```
@@ -563,6 +563,142 @@ tr:hover td { background: rgba(0,0,0,0.02); }
+
+.form-group.full-width {
+  grid-column: 1 / -1;
+}
+
+.inline-calc-wrapper {
+  grid-column: 1 / -1;
+  width: 100%;
+  animation: slideDown 0.3s ease-out;
+}
+
+@keyframes slideDown {
```

Stylelint (17.7.0)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)